lib/route.js

```diff
@@ -58,6 +58,7 @@ const { buildErrorHandler } = require('./error-handler')
 const { createChildLogger } = require('./logger')
 
 function buildRouting (options) {
+  options.config.useSemicolonDelimiter = true // To avoid a breaking change
```

Comment:
Cant we make this an options to be passed?
Solve it maybe like this?

```suggestion
  options.config.useSemicolonDelimiter ?= true // To avoid a breaking change
```

+ typescript typings and documentation?

Reply:
I currently have no time to add an option with all the things that it entails (TS and docs).

This is essentially to allow us to bump to the latest version of FMW without a breaking change.

Reply:
Imho create an issue and put a good-first-issud label is enough

Reply:
done